docs/reference/api/storage/ProfilingBlockDevice.md

```diff
@@ -1,2 +1,15 @@
 ## ProfilingBlockDevice
 
+The ProfilingBlockDevice class provides a BlockDevice implementation to wrap around an existing block device object to log reads, writes and erases.
+
+ProfilingBlockDevices take in a pointer to the block device being profiled as the only configurable parameter. All block device operations that wish to be logged need to be performed through the ProfilingBlockDevice object.
```

Comment:
Query: What does it mean that the operations need to be performed through the object? Please clarify. Also, please stop personifying operations. They don't like it when you do that.

Reply:
In order for read, write, and erase calls to be counted, they need to be be called from the ProfilingBlockDevice object, not the underlying block device directly. 
i.e. given a block device object `bd`, and a profilingblockdevice object `profiler` pointing to `bd` you'd need to call 
```c++
profiler.read(....);
```
rather than
```c++
bd.read(...);
```

Reply:
Ah. Makes sense. Please do. Thanks. 👍